src/util/Parameters.h

```diff
@@ -177,6 +177,7 @@ using String = Parameter<std::string, std::identity, std::identity, Name>;
 template <ParameterName Name>
 using MemorySizeParameter =
     Parameter<MemorySize, MemorySizeFromString, MemorySizeToString, Name>;
+
```

Comment:
Has nothing to do with this PR.
```suggestion
```

Reply:
Done, see [c24b87c](https://github.com/ad-freiburg/qlever/pull/1112/commits/c24b87c875cf97da71dc2de49a26f67c659e5f5f).